src/style/_type.scss

```diff
@@ -40,3 +40,9 @@
     font-size: $size;
     color: $color;
 }
+
+h1, h2, h3, h4, h5, h6, p, pre, label {
+    margin: 0;
+    padding: 0;
+    font-size: 100%;
```

Comment:
User agent style sheet imposes default font-sizes on text-elements - where one of these contains a child with `em` values the child will reference the user-agent font-size before our root level `62.5%`

Reply:
@Rody-Kirwan the intent here is to inherit the parent element's size at 100% instead?

Reply:
@shockey Yes - If we ensure that `font-size` is only applied to the lowest possible child elements in a given branch then `100%` here should always refer to the `62.5%` set in `.swagger-ui` i.e `10px`.